ocs_sample_library_preview/Users.py

```diff
@@ -0,0 +1,224 @@
+import json
+
+from .BaseClient import BaseClient
+from .Security import User
+from .Security import UserInvitation
+
+
+class Users(object):
+    """
+    Client for interacting with Users
+    """
+
+    AADIdentityProviderId = 'e2398938-bf8f-40fa-b380-d538ece2bfc2'
+    GoogleIdentityProviderId = 'd2a478ca-52e3-4fd4-9d93-ded440476364'
+    MicrosoftIdentityProviderId = '68a3a33b-b9dd-4eea-884f-87daefc4ff0f'
+
+    def __init__(self, client: BaseClient):
+        """
+        :param client: base client that handles auth and base routing
+        """
+        self.__tenant = client.tenant
+        self.__uri_api = client.uri_API
+        self.__base_client = client
+
+        self.__setPathAndQueryTemplates()
+
+    def getUsers(self, query: str = '', skip: int = 0, count: int = 100) -> list[User]:
+        """
+        Retrieves a list of users under the current tenant
+        :param query: filtering query
```

Comment:
is 'query' supported here? My tests in the API console says that it's not

Reply:
It is not. I have removed 'query' for this call.